util/webhook/webhook.go

```diff
@@ -189,11 +193,31 @@ func affectedRevisionInfo(payloadIf any) (webURLs []string, revision string, cha
 		// See: https://confluence.atlassian.com/bitbucket/event-payloads-740262817.html#EventPayloads-Push
 		// NOTE: this is untested
 		webURLs = append(webURLs, payload.Repository.Links.HTML.Href)
-		// TODO: bitbucket includes multiple changes as part of a single event.
-		// We only pick the first but need to consider how to handle multiple
-		for _, change := range payload.Push.Changes {
-			revision = change.New.Name
-			break
+		for _, changes := range payload.Push.Changes {
```

Comment:
I'd consider still taking just the first item. Better to avoid making two major behavior changes in the same PR.

Reply:
Processing only the first item in the change list to avoid any major behaviour change.